bundles/org.openhab.binding.vallox/src/main/resources/ESH-INF/thing/se-channels.xml

```diff
@@ -0,0 +1,547 @@
+		<state min="-20" max="40" step="1" pattern="%d °C" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="sePreHeatingSetPoint" advanced="true">
+		<item-type>Number</item-type>
+		<label>Pre heating set point</label>
+		<category>Temperature</category>
+		<state min="-20" max="40" step="1" pattern="%d °C" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seFilterGuardIndicator" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Filter Guard Indicator</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="sePostHeatingIndicator" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Post heating indicator</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seFaultIndicator" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Fault indicator</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seServiceReminderIndicator" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Service reminder indicator</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seHumidity" advanced="true">
+		<item-type>Number</item-type>
+		<label>Humidity</label>
+		<description>Highest measured relative humidity.</description>
+		<state min="0" max="255" step="1" pattern="%d %%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seBasicHumidityLevel" advanced="true">
+		<item-type>Number</item-type>
+		<label>Basic humidity level</label>
+		<description>When humidity adjust on ON, this is the target humidity what the machine tries to achieve</description>
+		<state min="0" max="100" step="1" pattern="%d %%" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seHumiditySensor1" advanced="true">
+		<item-type>Number</item-type>
+		<label>Humidity sensor 1</label>
+		<description>Relative Humidity measured from sensor 1</description>
+		<state min="0" max="255" step="1" pattern="%d %%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seHumiditySensor2" advanced="true">
+		<item-type>Number</item-type>
+		<label>Humidity sensor 2</label>
+		<description>Relative Humidity measured from sensor 2</description>
+		<state min="0" max="255" step="1" pattern="%d %%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2" advanced="true">
+		<item-type>Number</item-type>
+		<label>Measured Co2</label>
+		<state min="0" max="2000" step="1" pattern="%d ppm" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2High" advanced="true">
+		<item-type>Number</item-type>
+		<label>High byte of Co2 value</label>
+		<state min="0" max="255" step="1" pattern="%d" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2Low" advanced="true">
+		<item-type>Number</item-type>
+		<label>Low byte of Co2 value</label>
+		<state min="0" max="255" step="1" pattern="%d" readOnly="true"></state>
+	</channel-type>
```

Comment:
Why would you provide separate channels for each byte?

Reply:
Something that should've been removed long time ago. Removed.